components/automate-es-gateway/habitat/config/nginx.conf

```diff
@@ -72,7 +72,7 @@ http {
     client_body_buffer_size {{ cfg.ngx.http.client_body_buffer_size }};
     client_max_body_size {{cfg.ngx.http.client_max_body_size}};
 
-    proxy_set_header Host $http_host; # http_host preserves port # if present
+    proxy_set_header Host {{cfg.ngx.http.proxy_set_header_host}}; # http_host preserves port # if present
```

Comment:
[nit] I would prefer for this case to to have the default value for `proxy_set_header_host` be null and do an if else based on that. For example:
```
{{#if cfg.ngx.http.proxy_set_header_host }}
proxy_set_header Host {{cfg.ngx.http.proxy_set_header_host}};
{{ else }}
proxy_set_header Host $http_host;
{{/if}}
```

Reply:
will do!